### PR TITLE
[WIP] feat: added flag (--inspect)

### PR DESCRIPTION
**Pull request by Creatoon**
#### Short description of what this resolves:

With this pull request, you can look at all of your deployments on FAAS.

#### How it works:

1) Open your terminal & hit the command given below:-

```js
$ metacall-deploy  --inspect
```

#### Screenshots

WhatsApp Image 2022-02-18 at 10 11 13 PM

WhatsApp Image 2022-02-18 at 10 11 50 PM

- [x] My branch is up-to-date with the Upstream `master` branch.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -108,7 +110,12 @@ export const args = parse<CLIArgs>(
 			optional: true,
 			description: cliArgsDescription.plan
 		},
-		confDir: { type: String, alias: 'd', optional: true }
+		confDir: { type: String, alias: 'd', optional: true },
+		inspect: {
+			type: Boolean,
+			alias: 'i',
+			defaultValue: false
+		}
```
Comment (Siddhant-K-code) [inline src/index.ts:118]:
```suggestion
		inspect: { type: Boolean, alias: 'i', defaultValue: false }
```

**Review by Siddhant-K-code — COMMENTED** [review]
Marking `inspect` as optional or not.

@viferga what will be better way?

File: src/index.ts [hunk 2]
```diff
@@ -39,6 +40,7 @@ interface CLIArgs {
 	force: boolean;
 	plan?: Plans;
 	confDir?: string;
+	inspect: boolean;
```
[the remark this answers is not in the document]
Comment (Creatoon) [reply src/index.ts:43]: @Siddhant-K-code thanks for pointing this out, it should be optional.

File: src/index.ts [hunk 3]
```diff
@@ -108,7 +110,13 @@ export const args = parse<CLIArgs>(
...
 			description: cliArgsDescription.plan
 		},
-		confDir: { type: String, alias: 'd', optional: true }
+		confDir: { type: String, alias: 'd', optional: true },
+		inspect: {
+			type: Boolean,
+			alias: 'i',
+			defaultValue: false,
+			optional: true
+		}
```
Comment (Siddhant-K-code) [inline src/index.ts:119]: Keep Formatting same as #L113
```suggestion
		inspect: { type: Boolean, alias: 'i', defaultValue: false, optional: true }
```

**Review by Siddhant-K-code — CHANGES_REQUESTED** [review]
Just a Formatting Change

**Review by Siddhant-K-code — APPROVED** [review]
LGTM ✨ 

Thanks for the PR 🚀

**Comment by Creatoon** [comment]
> LGTM sparkles
> 
> Thanks for the PR rocket

Hey, this pr isn't completed yet, please look at the title [WIP]

**Pull request merged**